Failure: path must not be empty

Running `v1.2` on my repo gives an error `##[error]path must not be empty` and does not add the appropriate tags. I've set up the action and the config with almost identical configuration to what your have provided in examples. It worked a few times a week ago and now it is failing.

Can you provide some assisstance.

```
Run banyan/auto-label@1.2
##[debug]config: {"rules":{"patch":["*profile.info.yml"],"frontend":["*.js","*.scss","*.css","*.twig",".theme"],"backend":["*.php","*.module"],"ci":[".circleci","blt"]}}
##[debug]github.context.eventName: "pull_request"
##[debug]payload.action: "synchronize"
##[debug]result: {"repository":{"pullRequest":{"id":"MDExOlB1bGxSZXF1ZXN0MzgzNzc0ODE1","baseRefOid":"7bcb69a2ce19ca73d7a5fb1e96838b40f73e0cd5","headRefOid":"0008f129ef04b9a6ee02e8e665b19e2b3447bb53","baseRefName":"develop","headRefName":"8.2.4-release","labels":{"edges":[{"node":{"id":"MDU6TGFiZWwxODkwODk3ODI4","name":"patch"}}]}},"labels":{"edges":[{"node":{"id":"MDU6TGFiZWw5MDUzMjM2NTQ=","name":"bug"}},{"node":{"id":"MDU6TGFiZWw5MDUzMjM2NTU=","name":"duplicate"}},{"node":{"id":"MDU6TGFiZWw5MDUzMjM2NTY=","name":"enhancement"}},{"node":{"id":"MDU6TGFiZWw5MDUzMjM2NTc=","name":"help wanted"}},{"node":{"id":"MDU6TGFiZWw5MDUzMjM2NTg=","name":"good first issue"}},{"node":{"id":"MDU6TGFiZWw5MDUzMjM2NjE=","name":"invalid"}},{"node":{"id":"MDU6TGFiZWw5MDUzMjM2NjQ=","name":"question"}},{"node":{"id":"MDU6TGFiZWw5MDUzMjM2Njc=","name":"wontfix"}},{"node":{"id":"MDU6TGFiZWwxNzMwNjg2OTkw","name":"sparkbox feature branch"}},{"node":{"id":"MDU6TGFiZWwxODIwMjEyMjcx","name":"don't merge"}},{"node":{"id":"MDU6TGFiZWwxODkwODUwNzUx","name":"size/s"}},{"node":{"id":"MDU6TGFiZWwxODkwODc4MzEz","name":"ci"}},{"node":{"id":"MDU6TGFiZWwxODkwODk3ODI4","name":"patch"}},{"node":{"id":"MDU6TGFiZWwxODkwODk3ODMx","name":"release"}},{"node":{"id":"MDU6TGFiZWwxODkwOTM1NTAw","name":"minor"}},{"node":{"id":"MDU6TGFiZWwxODk4MTQ1MzEx","name":"frontend"}},{"node":{"id":"MDU6TGFiZWwxODk4MTQ1NzM5","name":"backend"}}]}},"rateLimit":{"limit":1000,"cost":1,"remaining":994,"resetAt":"2020-03-09T17:26:51Z"}}
##[debug]allLabels: {"bug":"MDU6TGFiZWw5MDUzMjM2NTQ=","duplicate":"MDU6TGFiZWw5MDUzMjM2NTU=","enhancement":"MDU6TGFiZWw5MDUzMjM2NTY=","help wanted":"MDU6TGFiZWw5MDUzMjM2NTc=","good first issue":"MDU6TGFiZWw5MDUzMjM2NTg=","invalid":"MDU6TGFiZWw5MDUzMjM2NjE=","question":"MDU6TGFiZWw5MDUzMjM2NjQ=","wontfix":"MDU6TGFiZWw5MDUzMjM2Njc=","sparkbox feature branch":"MDU6TGFiZWwxNzMwNjg2OTkw","don't merge":"MDU6TGFiZWwxODIwMjEyMjcx","size/s":"MDU6TGFiZWwxODkwODUwNzUx","ci":"MDU6TGFiZWwxODkwODc4MzEz","patch":"MDU6TGFiZWwxODkwODk3ODI4","release":"MDU6TGFiZWwxODkwODk3ODMx","minor":"MDU6TGFiZWwxODkwOTM1NTAw","frontend":"MDU6TGFiZWwxODk4MTQ1MzEx","backend":"MDU6TGFiZWwxODk4MTQ1NzM5"}
##[debug]currentLabelNames: ["patch"]
##[error]path must not be empty
##[debug]Node Action run completed with exit code 1
```

The action config is set up as:
```
name: PR Labeler
on:
  - pull_request

jobs:
  pr-labeler:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: banyan/auto-label@1.2
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

--- a/README.md
+++ b/README.md
@@ -23,6 +23,8 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 ```
 
+NOTE: `pull_request` event is triggered by [many actions](https://developer.github.com/v3/activity/events/types/#pullrequestevent), so please make sure to filter by `[opened, synchronize]` of [on.<event_name>.types](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onevent_nametypes) as in the example above.
+
 And configure by creating `.github/auto-label.json` file.
 The format is `label: rule`:
 

